translator/tools/wzdx_translator.py

```diff
@@ -1,17 +1,47 @@
-import xmltodict
-import uuid
-import sys
-import getopt
-from jsonschema import validate
-from jsonschema import ValidationError
 import logging
-from collections import OrderedDict
-import re
 import os
-from datetime import datetime
 import os.path
 import random
+import re
 import string
+import uuid
+from collections import OrderedDict
+from datetime import datetime, timezone
```

Comment:
```suggestion
from datetime import datetime
```
Looks like `timezone` isn't being used

Reply:
removed